public_sentiment_pipeline/load.py

```diff
@@ -0,0 +1,133 @@
+"""Contains the functions required to insert rows to the reddit_article table."""
+
+import sys
+import time
+
+from dotenv import dotenv_values
+from psycopg2 import connect
+
+from transform import run_transform
+
+REDDIT_TITLE_KEY = "title"
+REDDIT_SUBREDDIT_URL = "subreddit_url"
+REDDIT_ARTICLE_URL = "article_url"
+REDDIT_ARTICLE_DOMAIN = "article_domain"
+REDDIT_ARTICLE_SCORE = "score"
+REDDIT_UPVOTE_RATIO = "upvote_ratio"
+REDDIT_POST_COMMENTS = "comment_count"
+REDDIT_INCLUDED_COMMENTS = "included_comment_count"
+REDDIT_CREATED_UTC = "creation_timestamp"
+REDDIT_COMMENTS = "comments"
+REDDIT_SENTIMENT_MEAN = "mean_sentiment"
+REDDIT_SENTIMENT_ST_DEV = "st_dev_sentiment"
+REDDIT_SENTIMENT_MEDIAN = "median_sentiment"
+
+
+def establish_database_connection(config: dict):  # pragma: no cover
+    """Establishes a connection with the PostgreSQL RDS database."""
+    try:
+        return connect(
+            user=config['DATABASE_USERNAME'],
+            password=config['DATABASE_PASSWORD'],
+            host=config['DATABASE_ENDPOINT'],
+            port=config['DATABASE_PORT'],
+            database=config['DATABASE_NAME'])
```

Comment:
I've been using environ to retrieve my .env values so we may need to come together and decide what styles/formatting we'd all like to use to keep it consistent